nodes/src/nodes/llm_base/IInstance.py

```
@@ -21,37 +21,30 @@
-            case 'getContextLength':
-                return self.IGlobal._chat.getTotalTokens()
-            case 'getOutputLength':
-                return self.IGlobal._chat.getOutputTokens()
-            case 'getTokenCounter':
-                return self.IGlobal._chat.getTokens
-            case 'ask':
-                return self._question(param.question)
-            case _:
-                raise Exception(f'Invoke operation {param.op} is not defined')
+    @invoke_function
+    def getContextLength(self, param):
```

Ruff (0.15.7)

[warning] Unused method argument: `param`

(ARG002)